pkg/apply/processor/install.go

```diff
@@ -166,7 +167,7 @@ func (c *InstallProcessor) PreProcess(cluster *v2.Cluster) error {
 		cluster.SetMountImage(mount)
 		c.NewMounts = append(c.NewMounts, *mount)
 	}
-	runtime, err := runtime.NewDefaultRuntime(cluster, c.ClusterFile.GetKubeadmConfig())
+	runtime, err := kubernetes.NewDefaultRuntime(cluster, c.ClusterFile.GetKubeadmConfig())
```

Comment:
NewDefaultRuntime should be in `github.com/labring/sealos/pkg/runtime` .

Reply:
yep